packages/paste-theme/src/themeProvider.tsx

```diff
@@ -3,13 +3,15 @@ import styled from '@emotion/styled';
 import {Global, css, SerializedStyles} from '@emotion/core';
 import {themeGet} from '@styled-system/theme-get';
 import {ThemeProvider as StyledThemeProvider} from 'emotion-theming';
-import {DefaultTheme, SendGridTheme} from '@twilio-paste/theme-tokens';
+import {DefaultTheme, SendGridTheme, FlexTheme} from '@twilio-paste/theme-tokens';
 import {ThemeVariants} from './constants';
 
 interface GlobalStyleProps {
   theme: {};
 }
 const pasteGlobalStyles = (props: GlobalStyleProps): SerializedStyles => css`
+  @import url('https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i&display=swap');
```

Comment:
Do we have plans to load these only for the currently selected theme in the future?

Reply:
Certainly can. It only loads the font files for the font-face that is currently in use though too right? So it's not _dire_ right now

Reply:
I'm actually almost tempted to remove entirely and let font / asset handling be a product problem... just have a page on _what_ and _how_ to load them 🤔

Reply:
So this line in particular still hits the googleapis.com domain then loads that css.  The font files won't be loaded until use, correct.  Theoretically this means you can use any theme's fonts in any other theme, because the font-face rules are defined. Not a big deal, just an observation mainly.